### PR TITLE
Cancel warning modal

### PR DESCRIPTION
**Pull request by Corantin**
image

### CODE REVIEW

File: apps/web/components/CancelButton.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        btnStyle="outline"
+        color="danger"
+        onClick={() => setIsModalOpen(true)}
+      >
+        Cancel
+      </Button>
+      <Modal
+        isOpen={isModalOpen}
+        onClose={() => setIsModalOpen(false)}
+        title={`Cancel ${proposalData.title} #${proposalData.proposalNumber}`}
```
[the remark this answers is not in the document]
Comment (Corantin) [reply apps/web/components/CancelButton.tsx]: 16bd859743a8e10bc16f03bcde57bb77a7bda5d5

[the remark this answers is not in the document]
Comment (Corantin) [reply apps/web/components/CancelButton.tsx]: image

**Review by Lucianosc — APPROVED** [review]
LGTM

**Review by Mati0x — APPROVED** [review]
LG

**Pull request merged**